cafebabel/static/js/image-preview.js

```diff
@@ -16,7 +16,7 @@ fileInput &&
         if (canvasField) canvasField.classList.add('hidden')
         outputImage.innerHTML = `
     <figure>
-      <img src=${reader.result} alt="Picture preview">
+      <img src="${reader.result}" alt="Picture preview">
```

Comment:
Otherwise `jsmin` will remove the space before the `alt` and lead to wrong HTML…